core/src/test/java/org/infinispan/persistence/CacheAPIPeristenceTest.java

```diff
@@ -0,0 +1,51 @@
+package org.infinispan.persistence;
+
+import java.util.Collections;
+import java.util.Spliterators;
+
+import org.infinispan.api.APINonTxTest;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.container.DataContainer;
+import org.infinispan.container.DataContainer.ComputeAction;
+import org.infinispan.container.InternalEntryFactory;
+import org.infinispan.container.InternalEntryFactoryImpl;
+import org.infinispan.persistence.dummy.DummyInMemoryStoreConfigurationBuilder;
+import org.mockito.Mockito;
+
+/**
+ * Test that ensure that when persistence is used with an always empty data container that various operations
+ * are properly supported
+ * @author wburns
+ * @since 9.2
+ */
+public class CacheAPIPeristenceTest extends APINonTxTest {
+   @Override
+   protected void configure(ConfigurationBuilder builder) {
+      // We use a mocked container, so nothing is written
+      DataContainer mockContainer = Mockito.mock(DataContainer.class);
```

Comment:
I personally like adding static imports to avoir Mockito. noise

Reply:
Yeah I teeter back and forth. I will do that though.